tests/qos/test_buffer.py

```diff
@@ -17,7 +17,8 @@
 from tests.common.utilities import check_qos_db_fv_reference_with_table
 from tests.common.utilities import skip_release
 from tests.common.dualtor.dual_tor_utils import is_tunnel_qos_remap_enabled, dualtor_ports      # noqa F401
-from tests.qos.buffer_helpers import DutDbInfo
+from tests.qos.buffer_helpers import DutDbInfo, update_cable_len_for_all_ports    # noqa F401
```

Comment:
why is this change needed? Looks like update_cable_len_for_all_ports is not used in this file?

Reply:
Fixed. Thanks.